dashboard/src/actions/apps.ts

```diff
@@ -19,6 +28,10 @@ export const receiveAppList = createAction("RECEIVE_APP_LIST", resolve => {
   return (apps: IAppOverview[]) => resolve(apps);
 });
 
+export const receiveAppUpdateInfo = createAction("RECEIVE_APP_UPDATE_INFO", resolve => {
+  return (partialApp: { releaseName: string; updateInfo: IChartUpdateInfo }) => resolve(partialApp);
```

Comment:
`partialApp` is confusing, why not just `updateInfo`, or `payload` to keep it simple?

Reply:
Hmm I don't think I'm overthinking it. To me, `partialApp` sounds like it would have the type of `Partial<IAppOverview>`. In this we have a specific struct for the payload that includes just the updateInfo and releaseName. Thanks for changing it.